avocado/core/job.py

```diff
@@ -564,7 +570,6 @@ def run(self):
             self.exitcode |= exit_codes.AVOCADO_FAIL
             return self.exitcode
         finally:
-            self.job_pre_post_dispatcher.map_methods('post', self)
```

Comment:
Yes. BTW. It would be nice to define execution order for plugins.
My proposal is: results, archive, jobscript
results should go first for obvious reasons, but jobscript vs archive is not obvious, but IMHO jobscript should not allowed to add content to results dir, that is because archive can go second and jobscript third, In that case job script can perform mirroring of archived result dir to some job-store server.